news-bundle/src/Resources/contao/modules/ModuleNewsReader.php

```diff
@@ -134,29 +136,33 @@ protected function compile()
 		$arrArticle = $this->parseArticle($objArticle);
 		$this->Template->articles = $arrArticle;
 
-		// Overwrite the page title (see #2853, #4955 and #87)
-		if ($objArticle->pageTitle)
-		{
-			$objPage->pageTitle = $objArticle->pageTitle;
```

Comment:
for BC reasons, I think we should do this in addition to the response context

Reply:
I don't think that makes sense, no. It would keep generating deprecation notices.